kpops/utils/docstring.py

```diff
@@ -54,18 +51,10 @@ def _trim_description_end(desc: str) -> str:
     Works only with reStructuredText docstrings.
 
     :param desc: Description to be isolated, only the end will be trimmed
-    :type desc: str
+    :type desc: str, None
     :returns: Isolated description
     :rtype: str
     """
-    desc_enders: list[str] = [
-        ":param ",
-        ":type ",
-        ":return:",
-        ":rtype:",
-        "defaults to ",
-    ]
-
     try:
```

Comment:
if you make the `desc` a `str`, i.e. handle `None` cases outside this function (see comment above for reference), you can get rid of the whole try-except block